programs/vote/src/vote_state.rs

```diff
@@ -133,15 +148,6 @@ pub struct VoteState {
     /// the node that votes in this account
     pub node_pubkey: Pubkey,
 
-    /// the signer for vote transactions
```

Comment:
To preserve compatibility with 0.23.x, one option will be to create a `VoteState023` struct that matches what's on 0.23.x.   The `master` Vote program will then need to support deserializing `VoteState023` into `VoteState` in its process instruction and elsewhere.